store/store.go

```diff
@@ -109,3 +110,19 @@ func (Clock) Now() time.Time {
 func (Clock) After(d time.Duration) <-chan time.Time {
 	return time.After(d)
 }
+
+func initializeORM(config Config) *models.ORM {
+	var orm *models.ORM
+	var err error
+	sleeper := utils.NewConstantSleeper(config.DatabasePollInterval.Duration)
+	for {
+		orm, err = models.NewORM(path.Join(config.RootDir, "db.bolt"))
+		if err != nil && err.Error() == "timeout" {
```

Comment:
Last small thing: We try to match [the error](https://github.com/coreos/bbolt/blob/master/errors.go#L28) if possible, it's more exact than string comparison.

Something like `if err != nil && err == bolt.ErrTimeout {...` You might have to import bbolt as bolt: `bolt "github.com/coreos/bbolt"`